en/flight_modes_mc/position_slow.md

```diff
@@ -2,23 +2,33 @@
 
 [<img src="../../assets/site/difficulty_easy.png" title="Easy to fly" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
 
-_Position Slow_ is a variation of the regular [Position mode](../flight_modes_mc/position.md).
-It works the exact same way but with lower maximum velocities.
-This is useful to, at the flick of a mode switch enable more precision on the stick input, ensure to not fly too quickly close to obstacles or to comply with regulations like [EASA's low-speed mode/function](https://www.easa.europa.eu/en/light/topics/flying-drones-close-people).
+_Position Slow_ is a velocity and yaw rate limited version of the regular [Position mode](../flight_modes_mc/position.md).
 
-There are three ways to provide limits for slow down.
-The vehicle can only be limited to go slower than in _Position_ mode.
+The mode works in exactly the same way as _Position mode_ but with vehicle speed clamped to lower maximum velocities.
+You can use it to quickly slow down the vehicle to a safe speed (if it is moving faster than the maximum velocity in the limited axis).
+You can also use it to get more precision from stick input, in particular when flying close to obstacles, or to comply with regulations such as [EASA's low-speed mode/function](https://www.easa.europa.eu/en/light/topics/flying-drones-close-people).
+
+The velocity limits can be set using parameters, from a rotary knob or slider on an RC control, or using MAVLink.
+Limits set using RC control override those set by MAVLink, which in turn override those set using parameters.
+The limits can only be reduced below those for normal _Position_ mode.
 
 ## Set Limits using Parameters
 
-Once the vehicle is in _Position Slow_ mode it will slow down from the usual quicker velocities of _Position_ mode down to a slower set of defaults defined by parameters.
-For example, the horizontal maximum velocity by default slows down from 10 to 3m/s and the vertical one from 3 to 1m/s.
+The maximum values for slow mode horizontal velocity, vertical velocity, and yaw rate can be set using parameters.
+This approach is useful when the maximum desired speed in slow mode is fixed, and you just want to be able to quickly drop to a safer speed range (perhaps using a switch on your controller).
+
+The table below shows the parameters used to set the maximum values for _Position slow_ mode and _Position_ mode, respectively, along with their default values (in m/s and deg/s).
```

Comment:
Shouldn't we directly write the units in the table? That would be more clear to me.

Reply:
Yes :-).